### DISCUSSION TITLE
Ability to exclude/hide results by tag

### DISCUSSION THREAD

**Discussion by aolko** [civitai/civitai#195] [Ideas]
Subj.

**Comment by JustMaier** [comment]
Like this?
![image](https://user-images.githubusercontent.com/607609/215285646-ec4a5445-9c22-4e2d-95f8-94f94e7ba0da.png)

You have the ability to select any tag to hide from your account settings. You can also click the context menu on any card and click "Hide content with these tags":
![image](https://user-images.githubusercontent.com/607609/215285691-dbfde51d-7208-4153-9d4f-354ee39dd5a5.png)

Which brings up an options view like this:
![image](https://user-images.githubusercontent.com/607609/215285708-f7203a8a-c06e-4f2f-a9ed-8fd5497cd5c6.png)

Where you can pick which of the models tags you want to hide,

**Reply by aolko** [reply]
no, like this
![image](https://user-images.githubusercontent.com/581458/215286660-9ee0a2da-cbd0-427a-a2b7-a7d9e41aaf8b.png)
or this
![image](https://user-images.githubusercontent.com/581458/215286802-104d8986-f93c-4684-9b72-46b082c8fa26.png)


What you are showing is blacklisting a tag. I'm not a big fan of doing that, even if it comes to just searching.

**Reply by JustMaier** [reply]
Gotcha. We definitely have room for improvement when it comes to searching and filtering. I'll be sure to keep this in mind when we get around to improving that ❤️

**Comment by morphles** [comment]
Seems in two years this was note added ? :(

**Comment by drago87** [comment]
I'm still interested in this being added.
